neighboorhood/src/componenets/App.jsx

```diff
@@ -10,6 +15,24 @@ class App extends React.Component {
     };
   }
 
+  componentWillUpdate({ isScriptLoadSucceed }) {
```

Comment:
hey brian, not sure if think this is worth fixing but the componentWillUpdate function is deprecated. can you use a different method for this, perhaps componentDidUpdate?

Reply:
yes i can fix this thanks